packages/design-mui/src/components/MuiList/MuiList.tsx

```diff
@@ -130,24 +133,26 @@ const MuiListRows: React.FC<MuiListRowsProps> = ({
 
   return (
     <>
-      {data.map((item) => (
-        <TableRow key={item.id}>
-          {displays.map((display) => (
-            <TableCell key={`${item.id}-${display.key}`}>
-              {display.render({
-                record: item,
-              })}
-            </TableCell>
-          ))}
-        </TableRow>
-      ))}
+      {data.length === 0 && <EmptyList />}
```

Comment:
minor but lets turn this into a ternary I think, `data.length === 0 ? EmptyList : loop data`